scipy/stats/morestats.py

```diff
@@ -2721,50 +2722,109 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False):
         The second set of measurements.  If `y` is not given, then the `x`
         array is considered to be the differences between the two sets of
         measurements.
-    zero_method : string, {"pratt", "wilcox", "zsplit"}, optional
+    zero_method : {"pratt", "wilcox", "zsplit"}, optional
         "pratt":
-            Pratt treatment: includes zero-differences in the ranking process
-            (more conservative)
+            includes zero-differences in the ranking process,
+            but drop ranks of the zeros, see [4]_, (more conservative)
         "wilcox":
-            Wilcox treatment: discards all zero-differences
+            Wilcox treatment: discards all zero-differences, the default
         "zsplit":
             Zero rank split: just like Pratt, but splitting the zero rank
             between positive and negative ones
     correction : bool, optional
         If True, apply continuity correction by adjusting the Wilcoxon rank
         statistic by 0.5 towards the mean value when computing the
         z-statistic.  Default is False.
+    alternative : {"two-sided", "greater", "less"}, optional
+        The alternative hypothesis to be tested, see Notes. Default is
+        "two-sided".
 
     Returns
     -------
     statistic : float
-        The sum of the ranks of the differences above or below zero, whichever
-        is smaller.
+        If `alternative` is two-sided, the sum of the ranks of the
```

Comment:
Should two-sided be in quotes to indicate a string, to distinguish from two-sided referring to some general notion of the sense of the test?

Reply:
added quotes